### DISCUSSION TITLE
Message board/posting system

### DISCUSSION THREAD

**Discussion by lesleyspark** [redivis/meta#5] [Feature requests]
Two potential messaging system requests:
1. a way for dataset users to be able to talk to each other
2. a way for investigators to post classifieds "need a post doc to do x, y, z"

**Comment by sfmcintyre** [comment]
@vmeau and I discussed these today, some thoughts...
- slack is likely a great venue for workgroup-related and project-related discussion (about datasets related to a subject area, techniques, querying, resource sharing, etc)
- the data portal should decide how it facilitates the following: 
1. connecting people to the above slack in an easy way (maybe some integrations)
2. allowing for user -> admin communication
3. allowing for collaboration that isn't workgroup-related e.g. related to specific datasets, related to file groups
- with the classified-type posts, we should think about where they are initially generated (on a dataset, file group, project, etc) and where/how we should make them visible to people (feed, subscribe, etc)
